<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminCoachPage</class>
 <widget class="QWidget" name="AdminCoachPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Тренеры</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="topLayout">
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editButton">
       <property name="text">
        <string>Редактировать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportButton">
       <property name="text">
        <string>Экспорт в CSV</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchEdit">
       <property name="placeholderText">
        <string>Поиск тренера</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string>Найти</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="mainLayout">
     <item>
      <layout class="QVBoxLayout" name="inputFieldsLayout">
       <item>
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>Имя:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="birthLabel">
         <property name="text">
          <string>Дата рождения:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="birthDateEdit">
         <property name="displayFormat">
          <string>yyyy-MM-dd</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="jobLabel">
         <property name="text">
          <string>Должность:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="jobComboBox">
         <property name="editable">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>Главный тренер</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Помощник главного тренера</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Тренер вратарей</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Тренер по тактике</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Тренер по физподготовке</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="clubLabel">
         <property name="text">
          <string>Клуб:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="clubComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
